### PR TITLE
Add testing component for testing stuff

### PR DESCRIPTION
**Pull request by saleazer**
This pull request introduces a new component called `AppTestingComponent` and includes its integration into the application. The changes involve creating the component along with its template, adding unit tests, and updating the module declarations.

New component creation:

* [`src/app/app-testing/app-testing.component.ts`](diffhunk://#diff-5174a885576ab5277098070f01dfa1803a4df598502d8e4cb29aa28d91576636R1-R27): Created `AppTestingComponent` with methods `timesTwo`, `timesFour`, and `timesEight` to perform basic arithmetic operations.
* [`src/app/app-testing/app-testing.component.html`](diffhunk://#diff-404300d36185068c1250ec57ca078b000218e3ece9885747fd72eebe54cced76R1): Added a simple template for `AppTestingComponent` with a paragraph element.

Unit tests:

* [`src/app/app-testing/app-testing.component.spec.ts`](diffhunk://#diff-fc0e3eebd6d147dc7cf296361367db8647414a8bbd12444e79af688d46f61d88R1-R47): Added unit tests for `AppTestingComponent` to verify its creation and the correctness of the arithmetic methods.

Module integration:

* [`src/app/app.module.ts`](diffhunk://#diff-502d295b435b10ba99f5377504ca51149312e3cbebbbaa78ac5d925cb04cbfb0R14): Imported and declared `AppTestingComponent` in the module to ensure it is recognized and usable within the application. [[1]](diffhunk://#diff-502d295b435b10ba99f5377504ca51149312e3cbebbbaa78ac5d925cb04cbfb0R14) [[2]](diffhunk://#diff-502d295b435b10ba99f5377504ca51149312e3cbebbbaa78ac5d925cb04cbfb0L22-R24)

### CODE REVIEW

File: src/app/app-testing/app-testing.component.css
Comment (spacekat629) [inline src/app/app-testing/app-testing.component.css:1]: Do we need to have this file if it's empty?

Comment (saleazer) [reply src/app/app-testing/app-testing.component.css:1]: Probably not, I can get that removed.

File: src/app/app-testing/app-testing.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    .compileComponents();
+
+    fixture = TestBed.createComponent(AppTestingComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
```
Comment (spacekat629) [inline src/app/app-testing/app-testing.component.spec.ts:22]: This is a weak test, we prob don't need to leave this one in here..

File: src/app/app-testing/app-testing.component.html [hunk 2]
```diff
@@ -0,0 +1 @@
+<p>app-testing works!</p>
```
Comment (spacekat629) [inline src/app/app-testing/app-testing.component.html:1]: We should probably display something more useful on the actual component 😃

Comment (saleazer) [reply src/app/app-testing/app-testing.component.html:1]: Maybe later, lol

**Pull request merged**